trinity/sync/common/chain.py

```diff
@@ -64,12 +61,7 @@ def __init__(self,
                  db: AsyncHeaderDB,
                  peer_pool: PeerPool,
                  token: CancelToken = None) -> None:
-        self.complete_token = CancelToken('trinity.sync.common.BaseHeaderChainSyncer.SyncCompleted')
-        if token is None:
-            master_service_token = self.complete_token
-        else:
-            master_service_token = token.chain(self.complete_token)
-        super().__init__(master_service_token)
+        super().__init__(token)
```

Comment:
cc @gsalgado ^ I'm super distracted right now but pointing you at this location as a possible candidate for how #1234 is happening.

Reply:
I'd came across that yesterday when trying to find out if we were wrongly chaining tokens somewhere, but this actually looked correct to me